daml-lf/spec/contract-id.rst

```diff
@@ -45,18 +46,18 @@ Contract Identifiers
 A *contract identifier* (or contract ID for short) is a sequence of
 bytes defined as follows ::
 
-  ContractID := verisionPrefix ∥ discriminator ∥ suffix  
+  ContractID := versionPrefix ∥ discriminator ∥ suffix
 
 where
 
 * ``∥`` is the concatenation operation; 
-* ``verisionPrefix`` is 1 byte (equal to 0) used to version the
+* ``versionPrefix`` is 1 byte (equal to 0) used to version the
   contract ID scheme;
 * ``discriminator`` is a sequence of 32 bytes. It is like a random
   UUID, but generated from an initial seed (called *submission seed*)
   via a fixed `derivation scheme <Allocation scheme for
   discriminators_>`_ that supports validation (see below).
-* ``suffix`` is a sequence of 0 to 95 bytes used to enforce global
```

Comment:
In #5784 @hurryabit commented:
> Doesn't it say 96 in the DAML-LF 1 spec?

Reply:
The spec indeed said `96` before #5784, but it said after `94`. 
With `94` for the suffix, contract Id are at most 127 bytes long. 
This is the maximal length, to make the hexadecimal encoding less that 254 (the old limit for contract Id).
I fixed the 95.